homeassistant/util.py

```diff
@@ -22,6 +23,14 @@
 
 DATE_STR_FORMAT = "%H:%M:%S %d-%m-%Y"
 
+def f_to_c(temp):
+    temp = decimal.Decimal(temp)
```

Comment:
If we use float instead of decimal, we save a dependency. Float is less accurate but I think it doesn't matter in this case.